packaging/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+WD=$( pwd )
+# Run alembic migrations first
+( cd /code/node_types && alembic upgrade heads ) || \
+( cd /code/plugins && alembic upgrade heads ) || \
+( cd /code/blueprints && alembic upgrade heads ) || \
+( echo "Alembic upgrade failed" && exit 1 )
```

Comment:
You'll need here:
```suggestion

 ! { ( cd /code/node_types && alembic upgrade heads ) && \
 ( cd /code/plugins && alembic upgrade heads ) && \
 ( cd /code/blueprints && alembic upgrade heads ) } && \
 ( echo "Alembic upgrade failed" && exit 1 )
```
So that all 3 run and in case of failure the failed alert will run